Sources/MessagingInApp/Views/InAppMessageView.swift

```diff
@@ -191,6 +231,28 @@ public class InAppMessageView: UIView {
 
         runningHeightChangeAnimation?.startAnimation()
     }
+
+    // Takes in 2 Views. In 1 single animation, fades in 1 View while fading out the other.
+    private func animateCrossFade(fromView: UIView, toView: UIView, onComplete: (() -> Void)?) {
```

Comment:
Reading the name of the method, for a moment I thought this method animates when the "cross" button is tapped.
but the actual functioning is to animate in and out the inline views so how about `animateFadeInOutInlineView` for more clarity ?

Reply:
Done https://github.com/customerio/customerio-ios/pull/758/commits/76399b6fe2e8e6408968a0632fcbf9317979eedb